free-1.md

```diff
@@ -10,34 +10,48 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This DSL would help users learn more about flavor pairings and make suggestions if given specific foods. The users would be able to figure out foods or beverages that would pair well with a given item. For example, if a person were hosting a dinner party and didn't know what to serve with the main dish, they could receive suggestions about flavors or dishes that would be good accompaniments. 
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Flavor pairing is a unique domain and a DSL would help to educate users about successful pairings. It is appropriate for users because they would be able to interact with pairings in various ways (learning about more pairings, getting pairing suggestions, etc).
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I am very interested in food science and flavor combinations. I talked with one of my friends who has similar interests and she said that she didn't always know what flavors would pair well together when trying to create a meal or dish. I had looked for similar pairing services in the past and was not satisfied with my findings.
+
 
 ### Domain
 _Describe the project's domain in five words._
 
+Flavors that go well together.
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user could interact with the language by typing in a food or dish as well as what they wanted with it. They could specify that they wanted pairing suggestions in a certain category or they could ask for a random suggestion. The categories could be anything from dish vs. flavor to certain type of food. Perhaps the user wants a certain number of suggestions, perhaps the only want to know about beverages that would pair well. Since the language is designed for people to gain more information about food pairings, it makes sense for a program to revolve around a central object or dish. This way, the user could learn more about that specifc object. A more advanced feature of the language could be the ability to list multiple foods and receive suggestions common to all foods in the list.
+
```

Comment:
Maybe this might be thinking too far ahead, but it might better illustrate what exactly your language would be if you provided maybe a snippet of some code. I know I gave very high-level descriptions ( looks something like LateX), and definitely did not put as much thought as I will need to later.